src/Symfony/Component/Notifier/Bridge/Lox24/Lox24TransportFactory.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symfony\Component\Notifier\Bridge\Lox24;
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Symfony\Component\Notifier\Exception\UnsupportedSchemeException;
+use Symfony\Component\Notifier\Transport\AbstractTransportFactory;
+use Symfony\Component\Notifier\Transport\Dsn;
+
+/**
+ * @author Andrei Lebedev <andrew.lebedev@gmail.com>
+ */
+final class Lox24TransportFactory extends AbstractTransportFactory
+{
+    public function create(Dsn $dsn): Lox24Transport
+    {
+        $scheme = $dsn->getScheme();
+
+        if (!in_array($scheme, $this->getSupportedSchemes(), true))   {
+            throw new UnsupportedSchemeException($dsn, $scheme, $this->getSupportedSchemes());
+        }
+
+        $authUser = $this->getUser($dsn);
+        $authToken = $this->getPassword($dsn);
+        $from = $dsn->getRequiredOption('from');
+        $host = 'default' === $dsn->getHost() ? null : $dsn->getHost();
+        $port = $dsn->getPort();
+
+        return (new Lox24Transport(
+            "$authUser:$authToken",
```

Comment:
Please do not concat these values here, instead inject username and password to the transport and do it there